### PR TITLE
Header footer builder toprow testing

### PR DESCRIPTION
**Pull request by AtharvaAB8**

### CODE REVIEW

File: tests/RespTheme/ToprowCest.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+use \page\RespTheme\LogInAndLogOut;
+
+class ToprowCest
+{
+    public function _before(RespThemeTester $I)
+    {
+    }
+
+    // tests
+    public function tryToTest(RespThemeTester $I,  LogInAndLogOut $loginAndLogout, Customizer $customizer)
```
Comment (ratnesh-kadam) [inline tests/RespTheme/ToprowCest.php:13]: change the function name should be descriptive

File: tests/RespTheme/ToprowCest.php [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        
+        /*this checks the tablet layout settings for the top row */
+        $I->click('//*[@id="customize-footer-actions"]/div/div/button[2]');
+        $I->wait(2);
+        $I->selectOption('//*[@id="customize-control-responsive_header_tablet_top_layout"]/select', 'fullwidth');
+        $I->wait(2);
+        $I->selectOption('//*[@id="customize-control-responsive_header_tablet_top_layout"]/select', 'contained');
+        $I->wait(2);
+        $I->selectOption('//*[@id="customize-control-responsive_header_tablet_top_layout"]/select', 'standard');
+        $I->wait(2);
```
Comment (sonali-londhe) [inline tests/RespTheme/ToprowCest.php:42]: Use wait function only if necessary

File: tests/RespTheme/ToprowCest.php [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        $I->click('//*[@id="customize-control-responsive_top_row_background_desktop_color"]/label/div/div/div/div[2]/div/div/div[8]/button');
+        $I->wait(2);
+
+        /*this checks the tablet background colour for top row */
+        $I->click('//*[@id="customize-footer-actions"]/div/div/button[2]');
+        $I->scrollTo('//*[@id="customize-control-responsive_top_row_background_tablet_color"]');
+        $I->click('//*[@id="customize-control-responsive_top_row_background_tablet_color"]/label/div/div/button');
+        $I->wait(2);
+        //$I->click('//*[@id="customize-control-responsive_top_row_background_tablet_color"]/label/div/div/div/div[1]/div[2]/div[1]/button');
+        //$I->fillField('//*[@id="inspector-input-control-5"]', '#eeee22');
```
Comment (sonali-londhe) [inline tests/RespTheme/ToprowCest.php:89]: remove commented code

File: tests/RespTheme/ToprowCest.php [hunk 4]
```diff
@@ -0,0 +1,131 @@
+<?php
+
+use \page\RespTheme\Customizer;
+use \page\RespTheme\LogInAndLogOut;
+
+class ToprowCest
+{
+    public function _before(RespThemeTester $I)
```
Comment (sonali-londhe) [inline tests/RespTheme/ToprowCest.php:8]: remove empty functions

**Review by sonali-londhe — APPROVED** [review]

**Review by ratnesh-kadam — APPROVED** [review]

**Pull request merged**